util/regexp-assemble/lib/operators/comparer.py

```diff
@@ -0,0 +1,65 @@
+import sys
+from math import ceil
+
+from lib.operators.parser import Parser
+
+
+class Comparer(Parser):
+    def run(self, rule_id: str):
+        self.perform_compare_or_update(rule_id, self.compare_regex)
+
+    def run_all(self):
+        self.perform_compare_or_update(None, self.compare_regex)
+
+    def compare_regex(
+        self,
+        rule_id: str,
+        generated_regex: str,
+        current_regex: str,
+        config: dict,
+        config_key: str,
+    ):
+        if current_regex == generated_regex:
+            sys.stdout.write(f"Regex of {rule_id} has not changed")
+        else:
+            sys.stdout.write(f"Regex of {rule_id} has changed!")
```

Comment:
A `\n" at the end of the line would be fine.

Reply:
Done.